apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransactionCreationOptions.cs

```diff
@@ -183,6 +200,11 @@ internal TransactionOptions GetTransactionOptions()
         if (options is not null)
         {
             options.ExcludeTxnFromChangeStreams = ExcludeFromChangeStreams;
+            if(options.ReadWrite is not null)
+            {
+                // Options only settable on ReadWrite transactions (excluding the partioned dml txn as they too have TransactionMode set to ReadWrite)
```

Comment:
I think we can remove this, it's more or less clear from the code that this is an option for ReadWrite options only.